rainier-plot/src/main/scala/com/stripe/rainier/plot/Jupyter.scala

```diff
@@ -91,6 +93,95 @@ object Jupyter {
                             Some(xbounds))
     })
 
+  def whiskers[K, N](samples: Seq[Map[String, Double]]): Plot = {
+    val labels = samples.head.keys.toList.sorted
+    val data = labels.map { k =>
+      k -> samples.map { m =>
+        m(k)
+      }
+    }
+    whiskers(data, None)
+  }
+
+  def whiskers[K, N](seq: Seq[(K, Seq[N])], ybounds: Option[Bounds])(
+      implicit num: Numeric[N],
+      theme: Theme): Plot = {
+    val data = seq.map { case (_, ns)  => ns.map(num.toDouble) }
+    val labels = seq.map { case (k, _) => k.toString }
+
+    val xb = Bounds(0, seq.size.toDouble)
+    val yb = ybounds.getOrElse(
+      Bounds(
+        data.flatten.reduceOption[Double](math.min).getOrElse(0),
+        data.flatten.reduceOption[Double](math.max).getOrElse(0)
+      ))
+
+    val boxContexts = data.zipWithIndex.map {
+      case (dist, index) =>
+        if (dist.nonEmpty) {
+          val summary = BoxPlotSummaryStatistics(dist, (0.25, 0.50, 0.75))
```

Comment:
Should plausibly make these quantiles an optional argument as with https://github.com/cibotech/evilplot/blob/a44bdd84ceca4e198ff5527cf1aaed017febd91e/shared/src/main/scala/com/cibo/evilplot/numeric/BoxPlotSummaryStatistics.scala#L63